modules/google-speech/assets/config.schema.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
This file is auto-generated and should not be committed. Add it to gitignore please